mne/baseline.py

```diff
@@ -7,11 +7,19 @@
 
 import numpy as np
 
-from .utils import logger, verbose
+from .utils import logger
 
 
-@verbose
-def rescale(data, times, baseline, mode, verbose=None, copy=True):
```

Comment:
don't remove a public function just like that. I would keep it to avoid an unnecessary deprecation and deal with it with private calls.